sparta/test/TestingMacros.cmake

```diff
@@ -61,3 +60,15 @@ macro (sparta_recursive_copy build_target cp_file)
     add_custom_command (TARGET ${build_target} PRE_BUILD
       COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/${cp_file} ${CMAKE_CURRENT_BINARY_DIR}/)
 endmacro (sparta_recursive_copy)
+
+#
+# Create a sparta test executable
+#
+macro (sparta_add_executable target)
+  set (srcs "")
+  foreach(src_file ${ARGN})
+    list (APPEND srcs ${src_file})
+  endforeach()
+  add_executable (${target} ${srcs})
```

Comment:
Is this the same as:
```
add_executable(${target} ${ARGN})
```

Not that what you wrote is hard to understand, it's just a bit verbose if using `${ARGN}` like that will work.

Reply:
Done.  That's was stupid of me.